src/less/tooltip/base/tooltip.less

```diff
@@ -162,6 +162,66 @@ span.tourtip {
     width: @icon-width;
 }
 
+.dialog-infotip[role="dialog"] {
+    .dialog-base();
+}
+
+.dialog-infotip__window {
+    .dialog-window();
+
+    padding-bottom: 15vh;
+    top: 15%;
+    width: calc(100% - 32px);
+
+    &::before {
+        border-radius: 7px;
+    }
+}
+
+.dialog-infotip__window:not(.dialog-infotip__window--full) {
```

Comment:
This is a weird way to do this. Can you not do the `dialog-infotip__window--full` as a separate class and have those styles there? 
So the base should have all the styles it needs, and the full should have what it needs (and override if needed). 
Remember in CSS order matters to in terms of overriding styles. 

Also there is no full example